src/main/kotlin/view/InputView.kt

```diff
@@ -0,0 +1,68 @@
+package view
+
+import model.Lotto
+import model.UserBonusNumber
+import model.UserMainNumbers
+
+class InputView {
+    private fun readLineOrRetry(prompt: String): String {
+        while (true) {
+            print(prompt)
+            val input = readLine()
```

Comment:
```suggestion
            val input = readlnOrNull()
```

How about using the method with `null`? It might be safer.

Reply:
Yes, you're right, I will change that with step 2. :)